roles/pysk/tasks/main.yml

```diff
@@ -14,10 +14,10 @@
 ---
 - name: 'Install pysk dependencies'
   apt:
-    pkg: '{{item}}'
+    name: '{{ packages }}'
     state: latest
-  become: true
```

Comment:
We could maybe keep this `become: true` here as it doesn't really have anything to do with the apt changes?

Reply:
It's kinda redundant, innit? The top-level playbook file has `become: yes`, and the practice seems to be to set `become: false` in those roles/tasks that don't expect it.

But if you strongly prefer, I've no problem adding it back, of course.

Reply:
Yep, most roles practically assume that the whole playbook is run as `become: true` and I think it is fine. This just caught my eye as it looks like it just slipped away (e.g. the other task in the same role still has it).

I'd say that either let's add it back or remove `become` from the other task as well to leave the role in a consistent state. :)

Reply:
I'll add it back, better deal with the becomes in a separate PR (or at least a commit).